src/Nethermind/Nethermind.Config/INoCategoryConfig.cs

```diff
@@ -6,38 +6,38 @@ namespace Nethermind.Config;
-    public string DataDir { get; set; }
+    //[ConfigItem(Description = "Parent directory or path for BaseDbPath, KeyStoreDirectory, LogDirectory configurations.")]
+    //public string DataDir { get; set; }
 
-    [ConfigItem(Description = "Path to the JSON configuration file.")]
-    public string Config { get; set; }
+    //[ConfigItem(Description = "Path to the JSON configuration file.")]
+    //public string Config { get; set; }
 
-    [ConfigItem(Description = "Path or directory for configuration files.", DefaultValue = "configs")]
-    public string ConfigsDirectory { get; set; }
+    //[ConfigItem(Description = "Path or directory for configuration files.", DefaultValue = "configs")]
+    //public string ConfigsDirectory { get; set; }
 
-    [ConfigItem(Description = "Path or directory for database files.", DefaultValue = "db")]
-    public string BaseDbPath { get; set; }
+    //[ConfigItem(Description = "Path or directory for database files.", DefaultValue = "db")]
+    //public string BaseDbPath { get; set; }
 
-    [ConfigItem(Description = "Log level override. Possible values: OFF|TRACE|DEBUG|INFO|WARN|ERROR")]
-    public string Log { get; set; }
+    //[ConfigItem(Description = "Log level override. Possible values: OFF|TRACE|DEBUG|INFO|WARN|ERROR")]
+    //public string Log { get; set; }
 
-    [ConfigItem(Description = "Path to the NLog config file")]
-    public string LoggerConfigSource { get; set; }
+    //[ConfigItem(Description = "Path to the NLog config file")]
+    //public string LoggerConfigSource { get; set; }
 
-    [ConfigItem(Description = "Plugins directory")]
-    public string PluginsDirectory { get; set; }
+    //[ConfigItem(Description = "Plugins directory")]
+    //public string PluginsDirectory { get; set; }
 
     [ConfigItem(Description = "Sets the job name for metrics monitoring.", EnvironmentVariable = "NETHERMIND_MONITORING_JOB")]
     public string MonitoringJob { get; set; }
 
-    [ConfigItem(Description = "Sets the default group name for metrics monitoring.", EnvironmentVariable = "NETHERMIND_MONITORING_GROUP")]
-    public string MonitoringGroup { get; set; }
+    //[ConfigItem(Description = "Sets the default group name for metrics monitoring.", EnvironmentVariable = "NETHERMIND_MONITORING_GROUP")]
+    //public string MonitoringGroup { get; set; }
 
     [ConfigItem(Description = "Sets the external IP for the node.", EnvironmentVariable = "NETHERMIND_ENODE_IPADDRESS")]
     public string EnodeIpAddress { get; set; }
 
-    [ConfigItem(Description = "Enables Hive plugin used for executing Hive Ethereum Tests.", EnvironmentVariable = "NETHERMIND_HIVE_ENABLED", DefaultValue = "false")]
-    public bool HiveEnabled { get; set; }
+    //[ConfigItem(Description = "Enables Hive plugin used for executing Hive Ethereum Tests.", EnvironmentVariable = "NETHERMIND_HIVE_ENABLED", DefaultValue = "false")]
+    //public bool HiveEnabled { get; set; }
```

Comment:
@LukaszRozmej The `NETHERMIND_HIVE_ENABLED` seems redundant as we already have `NETHERMIND_HIVECONFIG_ENABLED` with `IHiveConfig.Enabled`.

Reply:
maybe